src/main/java/companySetUp/ConfigurationNewInformationTypePage.java

```diff
@@ -148,7 +148,7 @@ public void clickAccountTypeCreated() throws IOException {
 	 * 
 	 * @throws IOException
 	 */
-	public void enterAccountName() throws IOException {
+	public void clickAddInformationTypeButton() throws IOException {
 		ConfigurationNewInformationTypePage sp = new ConfigurationNewInformationTypePage(driver);
```

Comment:
@mayankchhajer  When you have changed the name of a method in a page then you also have to change the method name in their corresponding test class. This denotes you did not test the changes before commit and push.